python-threatexchange/threatexchange/exchanges/impl/ncmec_api.py

```diff
@@ -240,15 +242,18 @@ def fetch_iter(
                the cursor
         """
         start_time = 0
+        next_fetch = ""
```

Comment:
nit: just call this variable current_next_fetch - it's unconditionally written at 256.